llvm/tools/dsymutil/Options.td

```diff
@@ -218,3 +218,8 @@ def dsym_search_path: Separate<["-", "--"], "D">,
   MetaVarName<"<path>">,
   HelpText<"Specify a directory that contain dSYM files to search for.">,
   Group<grp_general>;
+
+def cas : Separate<["-", "--"], "cas">,
```

Comment:
Should we provide the additional options needed to configure plugin path and plugin options?